test/e2e/downloadsDisabled.test.ts

```diff
@@ -0,0 +1,26 @@
+import * as path from "path"
```

Comment:
Thoughts on putting this in the same file as the other download test?  My thinking is that they test the same code path so they should go together.

Reply:
I thought splitting out would help but that's anecdotal. I'll move it!

Reply:
Ohh very interesting point though, does Jest only run individual files in parallel?  I wonder if we can make it run `describe` blocks in parallel if so.

Reply:
Well this is being run by Playwright since it's an e2e test. It sounds like tests in the same file are not parallelized by default: https://playwright.dev/docs/test-parallel#parallelize-tests-in-a-single-file

> By default, tests in a single file are run in order. If you have 
many
 independent tests in a single file, you might want to run them in parallel with [test.describe.parallel(title, callback)](https://playwright.dev/docs/api/class-test#test-describe-parallel).

Reply:
Ooo that looks promising.

Reply:
I couldn't figure out the parallelism stuff. I may have misunderstood it from the beginning. I believe that it's for parallelizing `test`s within a `describe` block. But maybe it can be used for [parallelizing test suites as well](https://stackoverflow.com/questions/70100608/how-to-run-test-suite-in-parallel-but-not-the-single-tests)?

Either way, I'll move tests into one file. Then later if we have evidence the e2e tests are running slowly and we want to speed them up, we can look into it then.